mergify_engine/config.py

```diff
@@ -211,8 +211,8 @@ def AccountTokens(v: str) -> typing.Dict[str, str]:
 
 globals().update(Schema(CONFIG))
 
-if CONFIG["STREAM_URL"] is None:
-    STREAM_URL = CONFIG["STORAGE_URL"]
+if globals()["STREAM_URL"] is None:
```

Comment:
Why not using `config.get` or `in`?

Reply:
`STREAM_URL` is always set in `globals()` thanks to voluptuous and the L212